models/category-models.js

```diff
@@ -36,3 +36,12 @@ exports.fetchCommentsByReviewId = (review_id) => {
     })
 };
 
+exports.createCommentByReviewId = (review_id, body, username, date) => {
+    return db.query(`INSERT INTO comments (body, review_id, author, votes, created_at)
+                     VALUES ($1, $2, $3, $4, $5)
+                     RETURNING *`, [body, review_id, username, 0, date])
```

Comment:
If you look at the way that the tables are designed in `db/seeds/seed.js` then you will see that default values are automatically provided for both the `votes` column and the `created_at` column. So you actually don't need to set them up yourself (though what you've done totally works too!)

This is more of a suggestion for a refactor than a correction.